c31-swynlake/reactive-nutshell
# Bm login

# Description

Added load, login, and register files to authentication. From load, user is given links to either register or login. A login form or register form will then load. Verification processes make sure you can not register and account that already exists and that your username and password match in order to login. An option to Remember Me is given so that user can save their info to local storage instead of session storage if they wish.

Applies to issue # (6,7,14,15)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git getch --all`
2. `git checkout bm-login`
3. run `npm start`
4. Copy database.json.example into your database.json
4. Login using userName and password of existing user in your database. Punctuation matters. Verify that if you try to log in with incorrect info it will not let you. Make sure that your information is being saved to session or local storage depending on what you check. Also, verify that if you try and register with wither your existing username or email that it will not let you. Register a new user and make sure that it POSTS it to your database.json

# Checklist:
